build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 // TODO: Update group to end with 'admin' or 'trainee'.
-group = 'uk.nhs.hee.tis'
+group = 'uk.nhs.hee.trainee.forms'
```

Comment:
package should be `uk.nhs.hee.tis.trainee.forms` and the TODO needs to be removed.

Reply:
Make sure to update across the Java classes

Reply:
Made a mistake above and have now edited it.

Reply:
Changed accordingly